clarity_ext/service/file_service.py

```diff
@@ -265,7 +265,13 @@ def commit(self, disable_commits):
                                              if shared_file.id == artifact_id])
                     local_file = os.path.join(self.upload_queue_path, artifact_id, file_name)
                     self.logger.info("Uploading file {}".format(local_file))
-                    self.session.api.upload_new_file(artifact.api_resource, local_file)
+                    try:
+                        self.session.api.upload_new_file(artifact.api_resource, local_file)
+                    except requests.HTTPError as e:
+                        if "UNDER_REVIEW" in str(e):
+                            self.logger.error("Not able to upload step log as some of the samples are in review")
```

Comment:
I'm little confused here. If it's step log that is not working, then message to logger will not be shown to user (ok, from API though...). 

If it by any chance it's not the step log that causes problem, then this message might or might not be uploaded to server, depending on when step log is uploaded and when problem-file is uploaded.